common/libzkp/integration_test/testdata/phase1/proofs/README.md

```diff
@@ -0,0 +1,5 @@
+# Proof files would be automatically generated here
+
+### Some tests would try to generate the proofs it needed and cache them under this directory
+
+### Use `make clean-test-cache` to clean the cached proof files
```

Comment:
not sure whether it is outdated.  `make clean-test-cache`. reading.

Reply:
After discussion we would keep `integration-test` inside zkvm-prover, with new integration test being implied later in `scroll` side. So there would be no `intergration_test` directory in this PR